config/mappings.go

```diff
@@ -109,12 +109,17 @@ func HandleMappings(mappings []*legacyData.MappingDef, resolver resolve.Composit
 				}
 			}
 
-			if len(objMapping.targetFields) > 0 {
+			if len(objMapping.targetFields) > 1 {
```

Comment:
Why not just keep the > 0 and add `isObjectMapping = len(objMappingFIelds) > 1` instead of duplicating code

Reply:
Done